### DISCUSSION TITLE
Pretty Diffs - Visualize Ruff Changes

### DISCUSSION THREAD

**Discussion by juftin** [astral-sh/ruff#8284] [General]
[dunk](https://github.com/darrenburns/dunk) is a helpful CLI library to make git diffs pretty, I realized you can use it alongside ruff today - what an awesome way to visualize changes.

```shell
pipx install dunk
```

```shell
ruff . --diff | dunk
```
<img width="1894" alt="image" src="https://github.com/astral-sh/ruff/assets/49741340/8c057d2a-a5ab-406a-87be-9978e6fe280c">

```shell
ruff format --diff | dunk
```

<img width="1912" alt="image" src="https://github.com/astral-sh/ruff/assets/49741340/f645f08c-d967-495c-b6e0-ef5158fd6373">

**Comment by wu-clan** [comment]
Amazing!
